src/test/rustdoc/const-generics/const-impl.rs

```diff
@@ -33,7 +35,7 @@ impl<T> VSet<T, { Order::Unsorted }> {
 
 pub struct Escape<const S: &'static str>;
 
-// @has foo/struct.Escape.html '//div[@id="impl"]/h3[@class="code-header in-band"]' 'impl Escape<r#"<script>alert("Escape");</script>"#>'
+// has foo/struct.Escape.html '//div[@id="impl-Escape<r#"<script>alert("Escape")%3B</script>"#>"]/h3[@class="code-header in-band"]' 'impl Escape<r#"<script>alert("Escape");</script>"#>'
```

Comment:
The removal of the `@` means this test doesn't do anything. I think it was a typo on @jsha's part.

Reply:
That's correct, thanks for spotting that.